SDL/SDLMain.cpp

```diff
@@ -513,6 +513,10 @@ bool System_GetPropertyBool(SystemProperty prop) {
 #endif
 	case SYSPROP_HAS_BACK_BUTTON:
 		return true;
+#if PPSSPP_PLATFORM(SWITCH)
+	case SYSPROP_HAS_TEXT_INPUT_DIALOG:
+		return true;
```

Comment:
Technically should check `__nx_applet_type`.

-[Unknown]

Reply:
`__nx_applet_type == AppletType_Application` ?

Reply:
Well, just because it insta-fails checking that:
https://github.com/hrydgard/ppsspp/blob/a67991571d7a74f805527cedd39c8d673b9a9d97/SDL/SDLMain.cpp#L210-L211

-[Unknown]

Reply:
Oh duh, fixing.